api-docs/rst/dev-guide/api-operations/methods/put-update-policy-v1.0-tenantid-groups-groupid-policies-policyid.rst

```diff
@@ -189,20 +189,20 @@ This table shows the body parameters for the request:
 |                            |             |exclusive with the ``cron`` parameter.     |
 +----------------------------+-------------+-------------------------------------------+
 |scalingPolicies.[*].\       |Number       |The percent change to make in the number   |
-|**changePercent**           |*(Optional)* |of servers in the scaling group. If this   |
-|                            |             |number is positive, the number of servers  |
+|**changePercent**           |*(Optional)* |of units in the scaling group. If this     |
+|                            |             |number is positive, the number of units    |
 |                            |             |increases by the given percentage. If this |
 |                            |             |parameter is set to a negative number, the |
-|                            |             |number of servers decreases by the given   |
+|                            |             |number of units decreases by the given     |
 |                            |             |percentage. The absolute change in the     |
-|                            |             |number of servers is rounded to the        |
+|                            |             |number of units is rounded to the          |
 |                            |             |nearest integer. This means that if -X% of |
-|                            |             |the current number of servers translates   |
-|                            |             |to -0.5 or -0.25 or -0.75 servers, the     |
-|                            |             |actual number of servers that are shut     |
+|                            |             |the current number of units translates     |
+|                            |             |to -0.5 or -0.25 or -0.75 units, the       |
+|                            |             |actual number of units that are shut       |
 |                            |             |down is 1. If X% of the current number of  |
```

Comment:
New paragraph after "nearest integer."  

Change "This means..." ==> For example, if -X% of the current number translates to -0.5, -0.25, or -0.75 units, the actual number of units shut down is 1.  If ....